concat.js

```diff
@@ -125,18 +140,42 @@ Concat.prototype.build = function() {
       beginSection();
       concat.addSpace(this.footer + '\n');
     }
+
+    this._lastTree = this.getFSTree();
   }, this);
 };
 
+Concat.prototype.getFSTree = function() {
```

Comment:
thoughts on naming this `getCurrentTree` or `getCurrentFSTree`?

This would make line 87 self descriptive.

```js
var patch = this._lastTree.calculatePatch(this.getFSTree());
```